contracts/controller/MarketEnabled.sol

```diff
@@ -8,11 +8,13 @@ contract MarketEnabled is NutzEnabled {
   // address of the pull payemnt satelite
   address public pullAddr;
 
-  function MarketEnabled(address _pullAddr, address _storageAddr, address _nutzAddr) 
+  function MarketEnabled(address _pullAddr, address _storageAddr, address _nutzAddr)
     NutzEnabled(_nutzAddr, _storageAddr) {
     pullAddr = _pullAddr;
   }
 
+  uint256 internal INFINITY = 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff;
```

Comment:
nice, didn't know that variables can be internal.
there is no way to mark that they are constants, is there?